test/relative-time.js

```diff
@@ -159,4 +159,22 @@ suite('relative-time', function() {
     }
     assert.equal(root.children[0].textContent, 'now')
   })
+
+  test('rewrites given lang attribute', function() {
+    const now = new Date(Date.now() - 3 * 60 * 60 * 24 * 1000).toISOString()
+    const time = document.createElement('relative-time')
+    time.setAttribute('datetime', now)
+    time.setAttribute('lang', 'es')
```

Comment:
For this to work, maybe you would have to add `lang` to `observedAttributes`

Reply:
Done. Though, one issue here is that we can't easily observe the attributes of all our parents. If the document body's lang attribute changed, our children won't notice.